unity-renderer/Assets/Scripts/MainScripts/DCL/FeatureFlags/FeatureFlagConfiguration.cs

```diff
@@ -0,0 +1,22 @@
+using DCL;
+using UnityEngine;
+
+public class FeatureFlagController : IFeatureFlagController
+{
+    internal FeatureFlagBridge featureFlagBridgeComponent;
+
+    public FeatureFlagController()
+    {
+        if (InitialSceneReferences.i.bridgeGameObject != null)
+            AddBridgeComponent(InitialSceneReferences.i.bridgeGameObject );
```

Comment:
```suggestion
        if (InitialSceneReferences.i.bridgeGameObject != null)
            AddBridgeComponent(InitialSceneReferences.i.bridgeGameObject );
```
To avoid having a dependency on `InitialSceneReferences` maybe we can just inject the `bridgeGameObject` through the constructor?

Reply:
I don't think that this is a good idea because the FeatureFlagController is created in the PlatformContextFactory, and he should reference the InitialSceneReference making all the systems that reference the PlatafromContextFactory not being able to reference the InitialSceneRereference making the references a clusterfuck

I agree that it should be injected in the future when the refactor happen but as now I will let it here